x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/package-info.java

```diff
@@ -38,9 +38,11 @@
  *         Open Elasticsearch in IntelliJ.
  *     </li>
  *     <li>
- *         Open {@code x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/CsvTests.java}
- *         and run it. IntelliJ will take a few minutes to compile everything but the test itself
- *         should take only a few seconds. This is a fast path to running ESQL's integration tests.
+ *         Run the csv tests (see {@code x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/CsvTests.java})
+ *         from within Intellij or, alternatively, via Gradle:
+ *         {@code ./gradlew -p x-pack/plugin/esql test --tests "org.elasticsearch.xpack.esql.CsvTests"}
```

Comment:
Added this because, at least on my Windows, running from within Intellij IDEA does not work for some classpath/dependency resolution reasons.